Glyssen/Project.cs

```diff
@@ -86,9 +85,14 @@ private Project(GlyssenDblTextMetadata metadata, string recordingProjectName = n
 		}
 
 		public Project(GlyssenBundle bundle, string recordingProjectName = null, Project projectBeingUpdated = null) :
-			this(bundle.Metadata, recordingProjectName)
+			this(bundle.Metadata, recordingProjectName, false, bundle.WritingSystemDefinition)
 		{
 			Directory.CreateDirectory(ProjectFolder);
+			if (bundle.WritingSystemDefinition != null && bundle.WritingSystemDefinition.QuotationMarks != null && bundle.WritingSystemDefinition.QuotationMarks.Any())
```

Comment:
I think it would be better to move ConvertContinuersToParatextAssumptions into Project because it's not obvious that the WritingSystemDefinition in the bundle is the same object as the one pointed to by m_wsDefnition here.